### PR TITLE
Feat Disable Clamav

### PR DESCRIPTION
**Pull request by lohanidamodar**
## What does this PR do?

Disables ClamAV for newly installing users

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: app/config/variables.php [hunk 1]
```diff
@@ -311,7 +311,7 @@
                 'name' => '_APP_STORAGE_ANTIVIRUS',
                 'description' => 'This variable allows you to disable the internal anti-virus scans. This value is set to \'enabled\' by default, to cancel the scans set the value to \'disabled\'. When disabled, it\'s recommended to turn off the ClamAV container for better resource usage.',
```
Comment (eldadfux) [inline app/config/variables.php:312]: We need to update the default value in the description

**Review by eldadfux — CHANGES_REQUESTED** [review]
We also need to use the flag that disables the AV test on the Appwrite main container.

File: app/config/variables.php (hunk 1 above)
Comment (lohanidamodar) [reply app/config/variables.php]: Updated!

**Comment by lohanidamodar** [comment]
> We also need to use the flag that disables the AV test on the Appwrite main container.

Not sure I understand what you mean!

**Pull request merged**